PROJECT

```diff
@@ -1,9 +1,24 @@
+# Code generated by tool. DO NOT EDIT.
+# This file is used to track the info used to scaffold your project
+# and allow the plugins properly work.
+# More info: https://book.kubebuilder.io/reference/project-config.html
 domain: cray.hpe.com
 layout:
-  - go.kubebuilder.io/v4
+- go.kubebuilder.io/v4
 plugins:
   manifests.sdk.operatorframework.io/v2: {}
   scorecard.sdk.operatorframework.io/v2: {}
 projectName: nnf-dm
 repo: github.com/NearNodeFlash/nnf-dm
-version: '3'
+resources:
+- controller: true
+  domain: cray.hpe.com
+  group: nnfdm
```

Comment:
Is it not using this group for anything? This isn't the right group for these resources. The resources are part of nnf-sos, so group is just "nnf"

Reply:
This is defining a controller, not a resource.  But I see your point...however...

Here's why I didn't want to have "nnf" appear in this file as the group:
If we ever do create resources here, we should think carefully before we use the "nnf.cray.hpe.com" group+domain for them, given that they won't live with the other "nnf.cray.hpe.com" group+domain resources in nnf-sos. Think about how API versioning will be affected if there are two collections of "nnf.cray.hpe.com" resources, versioned independently in different repos...let's not. The tools I have for this are complicated enough, without going this way.

Reply:
I agree that any resources created in nnf-dm shouldn't use the "nnf" group. Do all of the entries listed under "resources" here have to have the same "group" though?

I don't think it really matters that the group is set to "nnfdm" because I don't think that's used for anything when there's only a controller and no resource definition. This file seems to be set up to assume that any controller in this project will also have a corresponding resource. That's not true here, so things get weird.

I'm fine leaving it as it is. I just commented because it wasn't what I was expecting.

Reply:
I think, when filling out the `kubebuilder create api` commandline args, there's a tendency to dig through the PROJECT file to figure out what to put in for some of the values.
At least, I have done that.